frontend/coprs_frontend/tests/test_models.py

```diff
@@ -100,7 +100,7 @@ def test_build_logs(self):
 
         def _pfxd(basename):
             pfx = ("http://copr-be-dev.cloud.fedoraproject.org/results/"
-                   "user1/foocopr/srpm-builds/00000001")
+                   "user1/foocopr/srpm-builds/00000001-foopackage")
```

Comment:
For srpm-builds -> I think we want to keep the old pattern.  The only problem we had with the old logic is non-determinism (it picks one or the other pattern depending on if the package name is known in advance, while it should _always_ use the `00000001` pattern).